doc/telestrat.sgml

```diff
@@ -0,0 +1,177 @@
+<!doctype linuxdoc system>
+
+<article>
+
+<title>Oric Telestrat-specific information for cc65
+<author>
+<url url="mailto:jede@oric.org" name="Jede">,<newline>
+
+<date>2017-01-22
+
+<abstract>
+An overview over the Telestrat (telemon 3.0 : http://orix.oric.org) runtime system as it is implemented for the cc65 C
+compiler.
+</abstract>
+
+<!-- Table of contents -->
+<toc>
+
+<!-- Begin the document -->
+
+<sect>Overview<p>
+
+This file contains an overview of the Telestrat runtime system as it comes with the
+cc65 C compiler. It describes the memory layout, Telemon3.0-specific header files,
+available drivers, and any pitfalls specific to that platform.
+
+Please note that Telemon3.0-specific functions are just mentioned here, they are
+described in detail in the separate <url url="funcref.html" name="function
+reference">. Even functions marked as "platform dependent" may be available on
+more than one platform. Please see the function reference for more
+information.
+
+
+
+<sect>Binary format<p>
+
+The standard binary output format generated by the linker for the Telemon 3.0 target
+is a machine language program with a 20 bytes header described here : http://orix.oric.org/doku.php?id=orix:header
```

Comment:
- For the first point, i am explaining a bit more in the sgml doc, and i'll update orix.oric.org for the header section.

- For the second point, yes it needs 2 headers, but it's not simple,  I updated sgml too, but it's a bit difficult on Telestrat for Telemon 2.4. 

When you have a Telemon 2.4 rom, you can't start any file from disk. Telemon 2.4 loads floppy disk boot sector and loads Stratsed Operating system from disk. In this way, the binary must be inserted on a Stratsed disk. But this tool does not exist, and should never exists because Oric international died before releasing docs. I would like to do it, but i have too many things to do before.
The only way is to use Floppy Builder from Osdk (http://osdk.defence-force.org/) to insert the binary sector per sector, track per track, but nobody will use this trick, because it's too complicated when you need to test your code. I explain this in the sgml doc.

That is why i created Orix and Telemon 3.0. I did Orix header like o65 format, but it's not the same : I would like to use o65 format in the future, and the best way, is to have similar header to share code.

I can manage Stratsed header (and i have less header doc for this one than Orix header). Let me know, but nobody will use Stratsed feature, because it's too much complicated to use it (mainly because there is a lack of tools)

- For the last point, there is no common header. I did my own header to manage 6502, 65C02, 65C816, and others stuffs. That is why i wanted to add telestrat target. There is 2000 Telestrat in the world, but, we are maybe 5 persons who are using the real hardware. The video on orix.oric.org is on emulator. If you want to see telestrat with telemon 3.0 and Orix working, here is the link : https://www.youtube.com/watch?v=TkzdlqF6-x8&list=PLugWQmG9T0gr9hSZNtmvDWkv2jpSwOW48